### ISSUE TITLE
Add rounded corners

### ISSUE THREAD

**Issue by SilverMarcs** [tkashkin/GameHub#560]
It would be nice to have rounded bottom corners for the app to better match gnome 40 and gtk4. Consistency :D

**Comment by shakeyourbunny** [comment]
No, if you wanna go backwards to Windows 2.0, they also had rounded corners.

Rounded corners are so 1982.
